### PR TITLE
tutum 0.21.1

### PR DESCRIPTION
**Pull request by ilovezfs**
Created with `brew bump-formula-pr`.

### CODE REVIEW

File: Formula/tutum.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 class Tutum < Formula
   desc "Docker platform for dev and ops"
   homepage "https://www.tutum.co/"
-  url "https://github.com/tutumcloud/cli/archive/v0.21.0.tar.gz"
-  sha256 "7657b337292cd480cd206776647bf6eee8c4e7b7ec370f42d2c2685818752e70"
+  url "https://github.com/tutumcloud/cli/archive/v0.21.1.tar.gz"
```
Comment (ilovezfs) [inline Formula/tutum.rb]: @DomT4 is there a preference between the GitHub archive and the pip download? They both seem to work equally well. I notice this one uses the GitHub archive and awscli uses the pip download.

Comment (DomT4) [inline Formula/tutum.rb]: I think the pip upload is supposed to be optimised to some degree, but I could be misremembering that from something else entirely. CC @tdsmith

Comment (ilovezfs) [inline Formula/tutum.rb]: For the sake of `brew liveupdate`, we probably should at least define a head url pointing to the GitHub repo if the stable url is going to point to a python url that cannot be parsed and the homepage is other than the GitHub repo as here.

Comment (DomT4) [inline Formula/tutum.rb]: If Tim's not around/tied up elsewhere/etc feel free to press ahead with the GitHub usage. It doesn't seem to hurt to use archive tarballs at least.

**Pull request closed without merging**